devicon.json

```
@@ -5776,6 +5776,22 @@
         },
         "color": "#294172", 
         "aliases": [] 
-    }
-
+    },
+    {
+    "name": "vitejs",
+    "tags": ["react", "compiler", "esbuild"],
```

Those aren't really relevant for vite, so here's some other suggestions
```suggestion
    "tags": [
        "tooling",
        "buildtool",
        "bundler",
        "javascript",
        "js",
    ],
```